contracts/protocore/Protocore.sol

```diff
@@ -51,4 +67,68 @@ contract Protocore is MosaicVersion, ValidatorSet {
     /* Storage */
 
     mapping(bytes32 /* vote message hash */ => Link) public links;
+
+    uint256 public openKernelHeight;
+    bytes32 public openKernelHash;
+
+
+    /* Special Functions */
+
+    /**
+     * @notice setup() function initializes the current contract.
+     *         The function will be called by inherited contracts.
+     *
+     * @param _coConsensus Address of the coconsensus contract.
+     */
+    function setup(
+        CoConsensusI _coConsensus
+    )
+        internal
+    {
+        CoConsensusModule.setupCoConsensus(_coConsensus);
```

Comment:
`CoConsensusModule.setupCoConsensus(_coConsensus);` will not be needed.
Maybe add a `todo` or `fixme` note, once https://github.com/mosaicdao/mosaic-1/issues/211 is merged, then this can be removed.